rebar.config

```diff
@@ -27,3 +27,8 @@
 ]}.
 
 {hex, [{doc, ex_doc}]}.
+
+{deps,
+ [
+  {eflame, "1.0.1"}
```

Comment:
eflame seems pretty cool! I didn't know about it until now.

... but can we put this and other dependencies (ex_doc?) under a specific profile (dev or test or something) so it isn't downloaded by default for random users?